tests/test_readme.py

```
@@ -0,0 +1,166 @@
+    def test_code_blocks_have_language_specification(self, readme_content: str) -> None:
+        """Test that code blocks specify language for syntax highlighting."""
+        content = readme_content
+        
+        # Find opening code blocks (ones followed by content)
+        lines = content.split('\n')
+        opening_blocks = []
+        
+        for i, line in enumerate(lines):
+            if line.startswith('```'):
+                # Check if this is an opening block (has content after it) or closing block
+                if i + 1 < len(lines) and lines[i + 1].strip():  # Next line has content
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)